gen/templates/custom_type.go.tmpl

```diff
@@ -0,0 +1,151 @@
+package customTypes
```

Comment:
What are your thoughts on  adding some extra test cases?

Reply:
I am adding one extra test for the customtype. 
Not sure, a negative test is useful for now, especially testing out of bounds values. We have a lot different attributes with a validator range that do not need a customtype. We just haven't implemented this yet in the resource schema. Hence we should add negative tests once it's implemented for all resources requiring a range validator and not only limited to customtype. It would be best to group these negative tests all in one PR.